pandas/tests/plotting/test_converter.py

```diff
@@ -91,6 +91,7 @@ def test_matplotlib_formatters(self):
                 assert Timestamp not in units.registry
             assert Timestamp in units.registry
 
+    @td.skip_if_lower_3_1_2_mpl
```

Comment:
sorry we don't really want to add this as a full decorator, just do this

``@td.skip_if_no("matplotlib", min_version="3.1.2")```